frontend/app/src/hocs/withScreencast/withScreencast.tsx

```diff
@@ -61,170 +54,164 @@ const log = getLogger("withScreencast")
 
 function withScreencast<P extends InjectedProps>(
   WrappedComponent: ComponentType<React.PropsWithChildren<P>>
-): ComponentType<React.PropsWithChildren<WrappedProps<P>>> {
-  class ComponentWithScreencast extends PureComponent<
-    WrappedProps<P>,
-    WithScreenCastState
-  > {
-    public static readonly displayName = `withScreencast(${
-      WrappedComponent.displayName || WrappedComponent.name
-    })`
-
-    private recorder?: ScreenCastRecorder | null
-
-    state = {
-      fileName: "streamlit-screencast",
-      recordAudio: false,
-      currentState: this.props.testOverride || ("OFF" as Steps),
-    }
+): React.FC<React.PropsWithChildren<WrappedProps<P>>> {
+  const ComponentWithScreencast: React.FC<
+    React.PropsWithChildren<WrappedProps<P>>
```

Comment:
nit: Can we just import FC, PropsWithChildren above in our react import?

Reply:
Changed it 👍